service/src/configuration.ts

```diff
@@ -83,10 +83,15 @@ export interface ConfigurationMetrics {
 
 export interface ConfigurationContent {
   /**
-   * The configuration for all the modules.
+   * The configuration for edge modules.
    */
   modulesContent?: {[key: string]: Object};
 
+  /**
+   * The configuration for device modules
+   */
+  moduleContent?: {[key: string]: Object};
```

Comment:
>moduleContent [](start = 2, length = 13)

<eye roll> seriously?  for edge modules, it's plural, for iothub modules, it's singular?  But the plural name still points to a singular object?  (I know this wasn't your choice.  I'm just making sure the name is as painfully dumb as it appears).